go/src/koding/klient/machine/index/change.go

```diff
@@ -103,6 +103,25 @@ func (cm *ChangeMeta) Coalesce(newer ChangeMeta) ChangeMeta {
 	}
 }
 
+// String implements fmt.Stringer interface and pretty prints stored change.
+func (cm *ChangeMeta) String() string {
+	cpy := atomic.LoadUint64((*uint64)(cm))
+
+	const cmstr = "LRaduH"
```

Comment:
I tried to use map approach but it has some drawbacks:

First I need to create `map[ChangeMeta]byte` but to preserve the order of iteration I will need to create another variable: slice `[]ChangeMeta` which I will be iterating over. This doesn't look very nice to me:

```go
var cm2byte := map[ChangeMeta]byte{
	ChangeMetaLocal: byte('L'),
	ChangeMetaRemote: byte('R'),
	ChangeMetaAdd: byte('a'),
	ChangeMetaRemove: byte('r'),
	ChangeMetaUpdate: byte('u'),
	ChangeMetaHuge: byte('H'),
}

var cmOrder = []ChangeMeta{
	ChangeMetaLocal,
	ChangeMetaRemote,
	ChangeMetaAdd,
	ChangeMetaRemove,
	ChangeMetaUpdate,
	ChangeMetaHuge,
}

// ...
for _, cm := range cmOrder {
   if cmOriginal&cm != 0 {
      // add cm2byte to return string, else "-" 
   }
}
```

Does this approach look better to you? Or am I missing something?